platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformBuilder.java

```diff
@@ -557,6 +575,10 @@ public PlatformComponentBuilder buildComponentBuilder() {
             throw new IllegalStateException("The current address book of the initial state is null.");
         }
 
+        if (bootstrapRoster == null) {
+            bootstrapRoster = createRoster(boostrapAddressBook);
```

Comment:
there are code coverage warnings on 404 and 579 for lack of coverage by tests.  Odd that 579 is untested.  Does that mean the null check is not matching in existing tests?

Reply:
resolved.